src/js/tables.js

```diff
@@ -49,13 +49,16 @@
 				$('[data-name]').each(function() {
 					var that = $(this),
```

Comment:
Why that, not self? Shouldn't it be Jq object already, you're iterating array of jq objects?